avm/ptn/azd/container-apps/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,77 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
```

Comment:
This is not an entirely accruate description as you're also providing optional values. I'd recommend to either make it a proper min test, or change the description to something else for fitting.